src/Bicep.Core/Semantics/Namespaces/K8sNamespaceType.cs

```diff
@@ -28,17 +30,17 @@ private static ObjectType GetConfigurationType()
             return new ObjectType("configuration", TypeSymbolValidationFlags.Default, new[]
             {
                 new NamedTypeProperty("namespace", LanguageConstants.String, TypePropertyFlags.Required, "The default Kubernetes namespace to deploy resources to."),
-                new NamedTypeProperty("kubeConfig", LanguageConstants.String, TypePropertyFlags.Required, "The Kubernetes configuration file, base-64 encoded."),
+                new NamedTypeProperty("kubeConfig", LanguageConstants.SecureString, TypePropertyFlags.Required, "The Kubernetes configuration file, base-64 encoded."),
```

Comment:
Not for this PR, but it'd be great to add a linter rule to block hard-coded secure config values later (similar to what we already have for secure parameters).